ios/FluentUI.Demo/FluentUI.Demo/Demos/PeoplePickerDemoController.swift

```diff
@@ -56,6 +56,7 @@ final class AsyncImageDemoPersona: PersonaData {
 
 class PeoplePickerDemoController: DemoController {
     var peoplePickers: [PeoplePicker] = []
+    var dividers: [MSFDivider] = []
```

Comment:
is there particular reason we need this?

Reply:
https://github.com/microsoft/fluentui-apple/pull/833#discussion_r770239258

Reply:
No longer needed now that we descend from UIView